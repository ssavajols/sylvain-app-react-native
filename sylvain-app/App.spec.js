import React from "react";
import TestRenderer from "react-test-renderer";

jest.mock("./assets/audio/sample.mp3", () => "mp3");

const App = require("./App").default;

describe("Root::App", () => {
  it("Should not crash", () => {
    const wrapper = TestRenderer.create(<App />);
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
