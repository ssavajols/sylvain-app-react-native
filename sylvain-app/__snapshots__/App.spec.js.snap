// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root::App Should not crash 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Open up App.js to start working on your app!
  </Text>
  <View
    accessibilityLabel="Learn more about this purple button"
    accessibilityRole="button"
    accessibilityStates={Array []}
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
            Object {
              "color": "#841584",
            },
          ]
        }
      >
        Play audio
      </Text>
    </View>
  </View>
</View>
`;
